CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Tamr VM Terraform Module
 
-## v4.3.0 - December 16th 2021
+
+## v4.4.0 - December 20th 2021
+* Deprecates `s3_policy_arns` in favor of `additional_policy_arns`.
+* Permits the ability to create the cloudwatch log group and pass it to the shell script.
```

Comment:
Replace with "Allows the creation of the Cloudwatch log group and passes it to the shell script"

Reply:
Done.